### PR TITLE
Set up and clean sphinx docs

### PR DESCRIPTION
**Pull request by fwmeng88**
We are going to use `readthedocs.io` to host the documentation.
This PR includes,
- [x] Add `.readthedocs.yml` to set up several configurations along with `requirements.txt` in `docs` folder
- [x] Fix indentation erros
- [x] Fix function display error for permutation Procrustes
- [x] Fix duplicate label problem
- [x] Update timestamp for all the documentations

### CODE REVIEW

**Comment by fwmeng88** [comment]
Now we have some updates for `README.md`

- [ ] Fix the dependencies
- [ ] Update version of 0.0.1alpha
- [ ] Add contact information
- [ ] Add badges for testing, documentationa and license

File: README.md [hunk 1]
```diff
@@ -21,16 +25,22 @@ Dependencies
...
-* Nosetests >= 1.3.7: http://readthedocs.org/docs/nose/en/latest/
+* PyTest >= 5.3.0: https://docs.pytest.org/
+* PyTest-Cov >= 2.8.0: https://pypi.org/project/pytest-cov/
 
 
 Installation
 ------------
 
-To install using package manager, run:
+Download the latest version of Procrustes with the following command,
```
Comment (msricher) [inline README.md]: end line with ":" not "," for consistency

File: docs/conf.py [hunk 2]
```diff
@@ -25,7 +25,7 @@
 # -*- coding: utf-8 -*-
 #
 # Procrustes documentation build configuration file, created by
-# sphinx-quickstart on Wed Apr 11 19:35:53 2018.
+# sphinx-quickstart on Wed Apr 11 19:35:53 2020.
 #
 # This file is execfile()d with the current directory set to its
```
Comment (msricher) [inline docs/conf.py]: what is "execfile()d"? Looks weird.

File: docs/conf.py [hunk 3]
```diff
@@ -89,17 +89,17 @@
...
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
 # The short X.Y version.
-version = '1.0.0'
+version = '0.0.1'
 # The full version, including alpha/beta/rc tags.
-release = '1.0.0'
+release = '0.0.1alpha'
```
Comment (msricher) [inline docs/conf.py]: release format should be like 0.0.0-alpha, with a hyphen

File: docs/usr_doc_chiral_check.rst [hunk 4]
```diff
@@ -31,7 +31,7 @@ procrustes method. Suppose we have two compounds with the same formula: CHFClBr
 chiral carbon atom in each of them. The absolute configuration of the compounds have been labeled in
 the figure.
 
-.. _lable:
+.. _lable2:
```
Comment (msricher) [inline docs/usr_doc_chiral_check.rst]: "label" not "lable"

File: docs/usr_doc_protein_align.rst [hunk 5]
```diff
@@ -37,7 +37,7 @@ inspired by the example in https://biomolecularstructures.readthedocs.io/en/late
 The structure is of cyclic - C2 global symmetry, where chain A (magenta) and chain C (green) are
 hemoglobin (deoxy) alpha chains, and chain B and D are hemoglobin (deoxy) beta chains.
 
-.. _lable:
+.. _lable1:
```
Comment (msricher) [inline docs/usr_doc_protein_align.rst]: "label"

File: docs/conf.py (hunk 2 above)
Comment (fwmeng88) [reply docs/conf.py]: Thank you for pointing it out. `execfile()d` by Sphinx with the current directory set to its containing dir. More details can be found in the first few paragraphs, at https://pythonhosted.org/CodeChat/CodeChat/template/conf.py.html.

File: README.md (hunk 1 above)
Comment (fwmeng88) [reply README.md]: Fixed in 6687ea2. Thank you for the suggestion.

File: docs/conf.py (hunk 3 above)
Comment (fwmeng88) [reply docs/conf.py]: Fixed in 6687ea2. Thank you for the careful inspection.

File: docs/usr_doc_chiral_check.rst (hunk 4 above)
Comment (fwmeng88) [reply docs/usr_doc_chiral_check.rst]: Fixed in 0e129a1. Thanks.

File: docs/usr_doc_protein_align.rst (hunk 5 above)
Comment (fwmeng88) [reply docs/usr_doc_protein_align.rst]: Fixed in 0e129a1. Thanks for pointing out the typo.

File: docs/examples/protein_alignment/2hhb.pdb [hunk 6]
```diff
@@ -2663,7 +2663,7 @@ ATOM   2014  N   THR B 123      24.217  -7.708  -5.373  1.00 17.94           N
 ATOM   2015  CA  THR B 123      23.829  -7.414  -6.760  1.00 23.37           C  
 ATOM   2016  C   THR B 123      22.296  -7.446  -6.917  1.00 24.37           C  
 ATOM   2017  O   THR B 123      21.620  -7.945  -6.037  1.00 15.04           O  
-ATOM   2018  CB  THR B 123      24.443  -8.449  -7.684  1.00 16.11           C  
+ATOM   2020  CB  THR B 123      24.443  -8.449  -7.684  1.00 16.11           C
```
Comment (FarnazH) [inline docs/examples/protein_alignment/2hhb.pdb]: I don't think this `2018` should be changed to `2020`.

File: docs/index.rst [hunk 7]
```diff
@@ -22,7 +22,7 @@
     : --
 
 .. Procrustes documentation master file, created by
-   sphinx-quickstart on Wed Apr 11 19:35:53 2018.
+   sphinx-quickstart on Wed Apr 11 19:35:53 2020.
```
Comment (FarnazH) [inline docs/index.rst]: This `2018` should not also be changed to `2020`.

File: README.md [hunk 8]
```diff
@@ -21,16 +25,22 @@ Dependencies
...
+* PyTest-Cov >= 2.8.0: https://pypi.org/project/pytest-cov/
 
 
 Installation
 ------------
 
-To install using package manager, run:
+Download the latest version of Procrustes with the following command.
+```bash
+    git clone git@github.com:theochem/chemtools.git
```
Comment (FarnazH) [inline README.md]: Change `chemtools` to `procrustes`.

File: README.md [hunk 9]
```diff
@@ -21,16 +25,22 @@ Dependencies
...
+```
+
+Then navigate to the Procrustes folder and run with package manager:
```
Comment (FarnazH) [inline README.md]: Instead of this sentence, I would use:
```bash
cd procrustes
```

File: docs/conf.py [hunk 10]
```diff
@@ -25,7 +25,7 @@
 # -*- coding: utf-8 -*-
 #
 # Procrustes documentation build configuration file, created by
-# sphinx-quickstart on Wed Apr 11 19:35:53 2018.
+# sphinx-quickstart on Wed Apr 11 19:35:53 2020.
```
Comment (FarnazH) [inline docs/conf.py]: I don't think this `2018` should be changed to `2020`.

File: docs/zreference.bib [hunk 11]
```diff
@@ -39,4 +39,4 @@ @article{zadeh2013molecular
   pages={927--936},
   year={2013},
   publisher={Springer}
-}
+}
```
Comment (FarnazH) [inline docs/zreference.bib]: I would add a new line at the end of the file for consistency.

Comment (fwmeng88) [reply docs/zreference.bib]: Newline has been added, 9195b51. It's good to follow a good convention. Thanks.

File: docs/conf.py (hunk 10 above)
Comment (fwmeng88) [reply docs/conf.py]: Problem fixed in b7e26dc. Thank for pointing this out.

File: README.md (hunk 9 above)
Comment (fwmeng88) [reply README.md]: This can make the installation clearer. Fixed in afe9d7f. Thanks.

File: README.md (hunk 8 above)
Comment (fwmeng88) [reply README.md]: Fixed typo in d0f9730. Thank you for reminding this.

File: docs/index.rst (hunk 7 above)
Comment (fwmeng88) [reply docs/index.rst]: Fixed in abc0c35.

File: docs/examples/protein_alignment/2hhb.pdb (hunk 6 above)
Comment (fwmeng88) [reply docs/examples/protein_alignment/2hhb.pdb]: Thank you for the careful reviewing. The problem has been fixed in 638d317.

**Comment by PaulWAyers** [comment]
Merged pull request so we can build docs and move forward. Can re-do later if problems arise.

**Pull request merged**